e2e/tests/functional/plugins/styling/stackedPlotStyling.e2e.spec.js

```diff
@@ -0,0 +1,211 @@
+  checkStyles,
+  hexToRGB,
+  setStyles
+} from '../../../../helper/stylingUtils.js';
+import { expect, test } from '../../../../pluginFixtures.js';
+
+const setBorderColor = '#ff00ff';
+const setBackgroundColor = '#5b0f00';
+const setTextColor = '#e6b8af';
+const defaultFrameBorderColor = '#e6b8af'; //default border color
+const defaultBorderTargetColor = '#aaaaaa';
+const defaultTextColor = '#aaaaaa'; // default text color
+const inheritedColor = '#aaaaaa'; // inherited from the body style
+const pukeGreen = '#6aa84f'; //Ugliest green known to man
+const NO_STYLE_RGBA = 'rgba(0, 0, 0, 0)'; //default background color value
+const setFontSize = '72px';
+const setFontType = 'sans-serif'; //Andale Mono", sans-serif
+
+test.describe('Stacked Plot styling', () => {
+  let stackedPlot;
+  test.beforeEach(async ({ page }) => {
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+    // Create a Stacked Plot
+    stackedPlot = await createDomainObjectWithDefaults(page, {
+      type: 'Stacked Plot',
+      name: 'StackedPlot1'
+    });
+
+    // Create two SWGs and attach them to the Stacked Plot
+    await createDomainObjectWithDefaults(page, {
+      type: 'Sine Wave Generator',
+      name: 'Sine Wave Generator 1',
+      parent: stackedPlot.uuid
+    });
+
+    await createDomainObjectWithDefaults(page, {
+      type: 'Sine Wave Generator',
+      name: 'Sine Wave Generator 2',
+      parent: stackedPlot.uuid
+    });
+  });
+
+  test('styling the overlay plot properly applies the styles to all containers', async ({
+    page
+  }) => {
+    // Directly navigate to the stackedplot
+    await page.goto(stackedPlot.url, { waitUntil: 'domcontentloaded' });
+
+    await page.getByLabel('Edit').click();
+
+    await page.getByRole('tab', { name: 'Styles' }).click();
+
+    await setStyles(
+      page,
+      setBorderColor,
+      setBackgroundColor,
+      setTextColor,
+      page.getByRole('tab', { name: 'Styles' }) //Workaround for https://github.com/nasa/openmct/issues/7229
+    );
+
+    //Set Font Size to 72
+    await page.getByLabel('Set Font Size').click();
+    await page.getByRole('menuitem', { name: '72px' }).click();
+
+    //Set Font Type to Monospace Bold
+    await page.getByLabel('Set Font Type').click();
+    await page.getByRole('menuitem', { name: 'Monospace Bold' }).click();
+
+    //Save
+    await page.getByRole('button', { name: 'Save' }).click();
+    await page.getByRole('listitem', { name: 'Save and Finish Editing' }).click();
+
+    await checkStyles(
+      hexToRGB(setBorderColor),
+      hexToRGB(setBackgroundColor),
+      hexToRGB(setTextColor),
+      page.getByLabel('Stacked Plot Style Target')
+    );
+
+    await checkFontStyles(setFontSize, setFontType, page.getByLabel('Stacked Plot Item Sine Wave Generator 1'));
+
+    // Reload page
+    await page.reload({ waitUntil: 'domcontentloaded' });
+
+    //Verify styles are correct after reload
+    await checkStyles(
+      hexToRGB(setBorderColor),
```

Comment:
This is returning another color entirely which makes me think we have some sort of bug. We don't need to fix it in this PR (and disable a portion of this test), but I'd like @khalidadil to take a look